authorization-server/routes/saml.js

```diff
@@ -0,0 +1,57 @@
+import { urlencoded } from 'express'; // eslint-disable-line import/no-unresolved
+import passport from 'passport';
+import { buildErrorMessage } from 'vite';
+import makeConfiguration from '../server-configuration.js';
+import Account from '../utils/account.js';
+import { cacheSubjectToken } from '../utils/id-token-cache.js';
+import { createSAMLStrategy } from '../utils/passport-strategy.js';
+
+const body = urlencoded({ extended: false });
+
+export default async (app, provider) => {
+  // TODO: The OIDC resource mounts this route and fails
+  if (!process.env.CUSTOMER1_SAML_ISSUER || process.env.CUSTOMER1_SAML_ISSUER === '') {
```

Comment:
nit, empty string is falsy
```suggestion
  if (!process.env.CUSTOMER1_SAML_ISSUER) {
```